.github/workflows/ci.yml

```diff
@@ -128,7 +136,7 @@ jobs:
             apt update && apt install -y --no-install-recommends ca-certificates cmake build-essential clang llvm git pkg-config autoconf automake libtool libelf-dev wget libb64-dev libc-ares-dev libcurl4-openssl-dev libssl-dev libtbb-dev libjq-dev libjsoncpp-dev libgrpc++-dev protobuf-compiler-grpc libgtest-dev libprotobuf-dev linux-headers-generic
 
           run: |
-            git config --global --add safe.directory ${{ github.workspace }}
+            git config --global --add safe.directory $GITHUB_WORKSPACE
```

Comment:
Switched to use `$GITHUB_WORKSPACE` per this bug: https://github.com/actions/checkout/issues/785

In this case we weren't affected, but since i used `$GITHUB_WORKSPACE` in other jobs, i decided to be consistent and use it everywhere.